### PR TITLE
added Landing page

### PR DESCRIPTION
**Pull request by bLAckBLAZER**
added Landing page along with NavBar, Footer and initial folder structure

### CODE REVIEW

File: src/components/Hero/Hero.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.hero-subtitle span {
+  color: var(--primary-color);
+}
+
+@media screen and (max-width: 640px) {
+  .grid-40-60 {
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
```
Comment (junaidshaikh-js) [inline src/components/Hero/Hero.css:31]: code comment should be removed

File: src/components/Hero/Hero.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          Manage your daily tasks and workflow in a modern way and boost your
+          efficiency without any efforts.
+        </div>
+        <Link to="/signup">
+          <button className="btn btn-primary mg-y-2">Sign Up</button>
+        </Link>
+        <Link to="/dashboard">
+          <a href="" className="link">
+            Already have an account? Log in!
+          </a>
```
Comment (junaidshaikh-js) [inline src/components/Hero/Hero.js:26]: You don't need to use anchor tag when using `Link` 

Link is the replacement for the `a` tag in react router.

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export const NavBar = ({ title, logo }) => {
+  return (
+    <nav className="navbar fixed">
+      <Link to="/">
+        <div className="flex align-ctr justify-ctr">
+          <div className="nav-logo">
+            <img src={logo} alt="" />
+          </div>
+          <div href="#" className="nav-heading">
```
Comment (junaidshaikh-js) [inline src/components/NavBar/NavBar.js:12]: unnecessary `href` attribute used

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            {title}
+          </div>
+        </div>
+      </Link>
+      <input type="text" className="input" placeholder="Search your to-do..." />
```
Comment (junaidshaikh-js) [inline src/components/NavBar/NavBar.js:17]: `input` element should have label for better accessibility.

**Review by junaidshaikh-js — COMMENTED** [review]
Great work! 

Good folder structure. Suggested changes can be done. 

When done you can merge the PR.

**Comment by bLAckBLAZER** [comment]
Thanks Junaid for such a detailed review. Really appreciate it. Made all the suggested changes.

File: src/components/Hero/Hero.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          Manage your daily tasks and workflow in a modern way and boost your
+          efficiency without any efforts.
+        </div>
+        <Link to="/signup">
+          <button className="btn btn-primary mg-y-2">Sign Up</button>
+        </Link>
+        <Link to="/dashboard">Already have an account? Log in!</Link>
+      </div>
+      <div className="hero-img flex-1">
+        <img src={HeroImage} alt="" className="img-res" />
```
Comment (shivampandey0) [inline src/components/Hero/Hero.js:26]: empty `alt` tag

**Review by shivampandey0 — COMMENTED** [review]
Clean and easy-to-understand code.
good to merge

File: src/components/Hero/Hero.js (hunk 5 above)
Comment (bLAckBLAZER) [reply src/components/Hero/Hero.js]: Thanks Shivam! made the changes

**Pull request merged**